ONSdigital/sdc-int-rh-ui
# Removed Google Analytics

# Motivation and Context
Google Analytics needed to be switched off while there were security concerns 

# What has changed
Removed Google Analytics

# How to test?
- Ran unit tests and it seems to work - but double check
- Check I haven't deleted too much/too little

# Links
https://trello.com/c/e5dGbqFb
